src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/GetAzureRmDataLakeAnalyticsCatalogItemAclEntry.cs

```diff
@@ -0,0 +1,120 @@
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.DataLakeAnalytics.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.DataLake.Analytics.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.DataLakeAnalytics.Commands
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmDataLakeAnalyticsCatalogItemAclEntry", DefaultParameterSetName = BaseCatalogParameterSetName),
+     OutputType(typeof(List<PSDataLakeAnalyticsAcl>), typeof(PSDataLakeAnalyticsAcl))]
+    [Alias("Get-AdlCatalogItemAclEntry")]
+    public class GetAzureRmDataLakeAnalyticsCatalogItemAclEntry : DataLakeAnalyticsCmdletBase
+    {
+        private const string BaseCatalogParameterSetName = "GetCatalogAclEntry";
+        private const string UserOwnerCatalogParameterSetName = "GetCatalogAclEntryForUserOwner";
+        private const string GroupOwnerCatalogParameterSetName = "GetCatalogAclEntryForGroupOwner";
+        private const string BaseCatalogItemParameterSetName = "GetCatalogItemAclEntry";
+        private const string UserOwnerCatalogItemParameterSetName = "GetCatalogItemAclEntryForUserOwner";
+        private const string GroupOwnerCatalogItemParameterSetName = "GetCatalogItemAclEntryForGroupOwner";
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            ParameterSetName = BaseCatalogParameterSetName, HelpMessage = "Specifies the Data Lake Analytics account name.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            ParameterSetName = UserOwnerCatalogParameterSetName, HelpMessage = "Specifies the Data Lake Analytics account name.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            ParameterSetName = GroupOwnerCatalogParameterSetName, HelpMessage = "Specifies the Data Lake Analytics account name.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            ParameterSetName = BaseCatalogItemParameterSetName, HelpMessage = "Specifies the Data Lake Analytics account name.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            ParameterSetName = UserOwnerCatalogItemParameterSetName, HelpMessage = "Specifies the Data Lake Analytics account name.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            ParameterSetName = GroupOwnerCatalogItemParameterSetName, HelpMessage = "Specifies the Data Lake Analytics account name.")]
+        [ValidateNotNullOrEmpty]
+        [Alias("AccountName")]
+        public string Account { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = true,
```

Comment:
@idear1203 you can remove the `ValueFromPipelineByPropertyName` and `Position` properties from parameters that are of type `SwitchParameter` (this comment applies to all files)